### DISCUSSION TITLE
JwtBearer configuration: Is't valid also for WEB API Auth?

### DISCUSSION THREAD

**Discussion by biapar** [marcominerva/SimpleAuthentication#108] [Q&A]
Is't the config schema valid also for WEB API Authorization when the frontend app pass the token ( access token or id token )?
The user login in the frontend app ( like NextJS ) and the pass the token to backend.
Thanks.

**Comment by marcominerva** [comment]
Excuse, I haven't understood your question. Can you explain the issue?
